packages/cli/src/commands/generate/scaffold/templates/components/EditNameCell.tsx.template

```diff
@@ -1,4 +1,9 @@
-import type { Edit${singularPascalName}ById } from 'types/graphql'
+import type {
+  ${singularPascalName},
+  Edit${singularPascalName}ById,
+  Update${singularPascalName}Input,
+  Update${singularPascalName}MutationVariables
+} from 'types/graphql'
 
 import { navigate, routes } from '@redwoodjs/router'
 import type { CellSuccessProps, CellFailureProps } from '@redwoodjs/web'
```

Comment:
This line gets removed when converting to JS, but then the import above get a red squiggle because of the empty line: 

<img width="653" alt="image" src="https://user-images.githubusercontent.com/300/177426570-615c6145-3879-46b0-b361-7cb33081129e.png">